### PR TITLE
Remove Old Construct Runtime Syntax

### PR DESCRIPTION
**Pull request by joepetrowski**
Test

- [x] Does not require a CHANGELOG entry

### CODE REVIEW

File: system-parachains/encointer/src/migrations_fix.rs [hunk 1]
```diff
@@ -22,7 +22,6 @@ pub mod balances {
 	use pallet_balances::*;
 	#[cfg(feature = "try-runtime")]
 	use sp_runtime::TryRuntimeError;
-	use sp_std::vec::Vec;
```
Comment (ggwpez) [inline system-parachains/encointer/src/migrations_fix.rs:25]: Looks like this is needed to make it compile.

**Review by ggwpez — APPROVED** [review]
I think its fine without Changelog.

File: system-parachains/encointer/src/migrations_fix.rs (hunk 1 above)
Comment (joepetrowski) [reply system-parachains/encointer/src/migrations_fix.rs]: Must be due to using `production` profile, with `dev` it gives an unused import warning. Added it back.

Comment (bkontur) [reply system-parachains/encointer/src/migrations_fix.rs]: `Vec` is used here only in the functions marked with `#[cfg(feature = "try-runtime")]`, so there are two solutions:

1. add feature before import:
```
#[cfg(feature = "try-runtime")]
use sp_std::vec::Vec;
```
2.  or remove import and use it directly:
```
fn pre_upgrade() -> Result<sp_std::vec::Vec<u8>
...
fn post_upgrade(_state: sp_std::vec::Vec<<u8>)
```

**Review by bkontur — APPROVED** [review]

**Review by kianenigma — APPROVED** [review]
Thanks, I hope we move to the beautiful https://github.com/paritytech/polkadot-sdk/pull/1378 soon.

**Review by bkchr — APPROVED** [review]

**Review by xlc — APPROVED** [review]

**Review by liamaharon — APPROVED** [review]

**Pull request merged**